macros/utils/table_operations/seed.sql

```diff
@@ -0,0 +1,21 @@
+{% macro seed_elementary_model(relation, csv_path) %}
+  {% do adapter.dispatch('seed_elementary_model', 'elementary')(relation, csv_path) %}
+{% endmacro %}
+
+{% macro default__seed_elementary_model(relation, csv_path) %}
+  {% set table = elementary.get_agate_table().from_csv(csv_path) %}
+  {% set model = elementary.get_node('model.elementary.' ~ relation.identifier.lower()) %}
+  {% do context.update({'this': relation}) %}
```

Comment:
Don't really like this.
This is why it's needed.

<img width="632" alt="image" src="https://user-images.githubusercontent.com/30181361/195567675-ec438144-90af-4724-ae0e-38b838e56cd5.png">